src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -376,6 +376,14 @@ The `<ViewTransitions />` router will trigger in-page transitions from `<form>`
 
 By default, your page will transition after a form is submitted, even if you are using `event.preventDefault()` to prevent this behavior.
 
+Please note that if your method is set to `post`, the default behavior of web browsers is to submit your form data encoded as as `application/x-www-form-urlencoded`, but the default behavior of Astro is to submit your form data as `multipart/form-data`. If you want to match the default behavior of web browsers make sure to set the `enctype` attribute accordingly:
+
+```astro title="src/components/Form.astro"
+<form action="/contact" method="post" enctype="application/x-www-form-urlencoded">
```

Comment:
```suggestion
<form action="/contact" method="POST" enctype="application/x-www-form-urlencoded">
```

Reply:
It's a stylistic thing. The docs currently have both. I'll leave it to you to decide :)

<img width="426" alt="image" src="https://github.com/withastro/docs/assets/174864/4fd50764-0a97-428f-ae3d-c5d30f56fabd">